source/frontend/src/components/maps/MapLeafletView.tsx

```diff
@@ -206,7 +219,7 @@ const MapLeafletView: React.FC<React.PropsWithChildren<MapLeafletViewProps>> = (
             ))}
           </LayerGroup>
         )}
-        {mapMachine.showPopup && (
+        {mapMachine.showPopup && !mapMachine.isRepositioning && (
```

Comment:
this is kinda odd. The mapMachine should be the sole authority in deciding if the popup should be shown. Otherwise, we risk having the logic distributed. Would it be possible to have the state so both showPopup and isRepositioning can never happen together?

Reply:
Updated